code/modules/organs/external/_external_icons.dm

```diff
@@ -145,9 +145,15 @@ GLOBAL_LIST_EMPTY(limb_icon_cache)
 
 /obj/item/organ/external/proc/get_icon(var/skeletal)
 
+	if(owner && ishuman(owner))
+		var/mob/living/carbon/human/H = owner
+		s_base = LAZYACCESS(species.base_skin_colours, H.s_base)
 	var/gender = "f"
 	if(owner && owner.gender == MALE)
 		gender = "m"
+	if(owner && owner.gender == FEMALE)
+		gender = "f"
```

Comment:
because i noticed the gender here wasnt changing when I adjusted the gender in char setup, and I thought it was some weird issue with being male and switching to female, so I added this, but it was the DNA stuff further down the line